map/src/components/resultDetail.tsx

```diff
@@ -0,0 +1,247 @@
+import { ContactDetails } from '@reach4help/model/lib/markers';
+import React from 'react';
+import { MdEmail, MdLanguage, MdPhone } from 'react-icons/md';
+import { MarkerIdAndInfo } from 'src/components/map';
+import { Language, t } from 'src/i18n';
+
+import styled from '../styling';
+import MarkerType from './marker-type';
+
+interface Props {
+  result: MarkerIdAndInfo;
+  lang: Language;
+  fullDetail?: boolean;
+}
+
+const Div = styled('div')`
+  overflow-y: auto;
+  max-height: 100%;
+  background: #fff;
+  padding: ${p => p.theme.spacingPx}px;
+  overflow-y: auto;
+  background: #fff;
+  box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.15);
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+  cursor: auto;
+  pointer-events: initial;
+
+  > .disclaimer,
+  > .source {
+    background: rgba(0, 0, 0, 0.05);
+    border-radius: 3px;
+    padding: 10px 8px;
+    color: rgba(0, 0, 0, 0.5);
+    margin-bottom: 20px;
+
+    strong {
+      text-transform: uppercase;
+    }
+  }
+
+  > .disclaimer {
+    font-size: 10px;
+    line-height: 16px;
+  }
+
+  > .source {
+    font-size: 12px;
+    line-height: 150%;
+  }
+
+  > .name {
+    font-size: 1.5rem;
+    padding-bottom: ${p => p.theme.spacingPx / 2}px;
+  }
+
+  > .location {
+    color: ${p => p.theme.textColorLight};
+    font-size: 0.9rem;
```

Comment:
Is there a reason this one is in `rem` and the other font sizes are in `px`? I feel it would be better to be consistent unless you have a specific reason to do it this way

Reply:
same here

Reply:
@jensren will change rem to px, seems like majority of the code is using px